rails_docker/docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - '3000:3000'
     volumes:
       - '.:/#{app_name}'
```

Comment:
The volumes config need to contain the bundle volume as well to avoid rebuilding it each time:

```
volumes:
      - '.:/#{app_name}'
      - "ruby-bundle:/bundle"
```